streaming/src/test/scala/org/apache/spark/streaming/StreamingContextSuite.scala

```diff
@@ -33,8 +33,12 @@ import org.apache.spark.storage.StorageLevel
 import org.apache.spark.streaming.dstream.DStream
 import org.apache.spark.streaming.receiver.Receiver
 import org.apache.spark.util.Utils
-import org.apache.spark.{Logging, SparkConf, SparkContext, SparkException, SparkFunSuite}
-
+import org.apache.spark.{Logging, SparkConf, SparkContext, SparkFunSuite}
```

Comment:
Again, your imports are not organized properly. Please pay careful attention to this. spark imports go together and are sorted alphabetically. scala imports should go together too.